src/components/timeline/index.css

```diff
@@ -65,40 +65,57 @@
 .timelineSettingsHiddenTracks {
   /* Make sure and override all rules that have to do with button defaults. */
   position: relative;
-  height: 17px;
-  padding: 0 7px 0 20px;
+
+  /* 1px is allocated for the vertical separator line. */
+  width: calc(var(--thread-label-column-width) - 1px);
+  height: 20px;
+
+  /* 8px is intentional to align this text with the text above. */
+  padding: 0 20px 0 8px;
   border: 0;
-  margin: 4px 5px 0;
-  background-color: var(--blue-60);
-  border-radius: 12px;
-  color: #fff;
+  margin: 0;
+  background: none;
+  color: inherit;
+
+  /* Specific for links. */
+  cursor: default;
   font: inherit;
-  line-height: 17px;
+  line-height: 20px;
+  text-align: left;
+  text-decoration: none;
```

Comment:
The text-decoration looks unnecessary too.

I suppose you copy pasted from somewhere where we used this for links? :)

Reply:
Yeah, I copy pasted it from the menu buttons. But they were links and this is a button. I guess this is the reason we don't need them here.